File: javascript/socketd/src/socketd/utils/SessionUtils.ts
import {ClientSession} from "../transport/client/ClientSession";

/**
 * 会话工具（主要检测状态）
 *
 * @author noear
 * @since 2.4
 */
export class SessionUtils {
    /**
     * 是否活动的
     */
    static isActive(s: ClientSession): boolean {
        return s != null && s.isActive();
    }

    /**
     * 是否有效的
     */
    static isValid(s: ClientSession): boolean {
        return s != null && s.isValid();
    }
}

File: python/socketd/socketd/utils/SessionUtils.py
from socketd.transport.client.ClientSession import ClientSession

# 会话工具（主要检测状态）
class SessionUtils:
    @staticmethod
    def is_active(s: ClientSession) -> bool:
        return s and  s.is_active()

    @staticmethod
    def is_valid(s: ClientSession) -> bool:
        return s and s.is_valid()